js/components/charts/chart.html

```diff
@@ -1,4 +1,4 @@
-<div data-bind="reference: setContainer, if: data"></div>
+<div data-bind="chart: $component, if: data"></div>
```

Comment:
too abstract. why not to define those 5 props explicitly

Reply:
Beacause it's not really nessessary if you pass here a component which extends Chart